### ISSUE TITLE
WebGL issue

### ISSUE THREAD

**Issue by be4zad** [sz3/libcimbar#95]
My GPU is old and doesn't much support GL related things. so the website doesn't show the barcode and this error appears in the console:
```
Failed to create WebGL context: WebGL creation failed: 
* tryNativeGL (FEATURE_FAILURE_EGL_NO_CONFIG)
* Exhausted GL driver options. (FEATURE_FAILURE_WEBGL_EXHAUSTED_DRIVERS)
```

My browser is firefox and the issue may solve by:

* Enabling hardware acceleration
* Toggling webgl.out-of-process in about:config (worked for me)
* Toggling webgl.force-enabled in about:config

It's good idea to handle the error and suggest to user to try above suggestions.

**Comment by sz3** [comment]
Yeah, error handling should be better. At the very *least* it'd be good to report the `WebGL creation failed` part back.

**Comment by sz3** [comment]
I believe the error reporting is better in the current version (https://github.com/sz3/libcimbar/releases/tag/v0.6.1f)
